sklearn/tests/test_cross_validation.py

```diff
@@ -370,7 +370,10 @@ def test_label_kfold():
     # Construct the test data
     tolerance = 0.05 * n_samples  # 5 percent error allowed
     labels = rng.randint(0, n_labels, n_samples)
```

Comment:
This test seems to easy to me: the labels are balanced, hence the rebalancing logic and it's interaction with shuffling is not properly evaluated. It would be better to have a couple of labels with 2 to 3 times more samples than the others for instance.